notification/package.json

```diff
@@ -43,7 +43,8 @@
     "jest": "^26.5.3",
     "ts-jest": "^26.4.1",
     "tslib": "^2.0.3",
-    "typescript": "^4.0.3"
+    "typescript": "^4.0.3",
+    "@jvalue/node-dry-basics": "0.0.3"
```

Comment:
You must move this from `devDependencies` to `dependencies`

Reply:
oh darn, I'm so stupid... Thanks a lot <3